python_modules/libraries/dagster-dbt/dagster_dbt/dagster_dbt_translator.py

```diff
@@ -110,6 +115,39 @@ def get_metadata(cls, dbt_resource_props: Mapping[str, Any]) -> Mapping[str, Any
         """
         return default_metadata_fn(dbt_resource_props)
 
+    @classmethod
+    def get_group_name(cls, dbt_resource_props: Mapping[str, Any]) -> Optional[str]:
```

Comment:
```suggestion
    def get_group(cls, dbt_resource_props: Mapping[str, Any]) -> Optional[str]:
```

Reply:
The property on `@asset` is `group_name`, so I thought it would be best to be consistent.